app/assets/stylesheets/application.css.scss

```diff
@@ -159,3 +159,8 @@ main {
   max-width: 150%;
   margin-left: 0px;
 }
+
+.avatar{
+  background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.